main.qml

```diff
@@ -794,4 +794,8 @@ ApplicationWindow {
             }
         }
     }
+    onClosing: {
+        walletManager.closeWallet(currentWallet);
+        console.log("onClosing called");
+    }
```

Comment:
@mbg033: FYI. Noticed that Wallet::~Wallet() wasn't called on some occasions when application was closed. Added these lines to make sure the wallet is saved properly. Have you noticed this behavior too?